package.json

```
@@ -12,16 +12,19 @@
 	"license": "Apache-2.0",
 	"private": true,
 	"scripts": {
+		"pretest": "yarn build",
 		"test": "jest --collect-coverage",
-		"test:ci": "jest --verbose --ci --no-stack-trace --collect-coverage",
+		"test:ci": "jest --verbose --ci --runInBand --no-stack-trace --collect-coverage",
```

```suggestion
		"test:ci": "yarn test --verbose --no-stack-trace",
```

- Jest automatically runs with `--ci` when `CI` env var is true, which it is for GH Actions
- `--runInBand` is only meant for debugging purposes. There is no real reason to run tests in serial instead of parallel in CI, especially a modern CI environment like GH actions which can handle worker pools just fine.
- Because yarn doesn't rely on the hacky `npm run test -- --verbose` thing to pass extra parameters we can apply some DRY coding here by calling the base `test` script.

_personally_ I would also remove the `--verbose` and `--no-stack-traces` options, thus just call `test` in CI, but if you want those flags enabled then that's your call.

---

`--runInBand` goes back to an awfully slow CI server I encountered in a certain professional environment.
My CI times were cut in half when it wasn't running parallel, but you might be right this is hosted on MS cloud here after all.